### DISCUSSION TITLE
Pull Shark achievement not showing all my merged PRs

### DISCUSSION THREAD

**Discussion by elidrissidev** [community/community#18305] [Other Feature Feedback, Questions, & Ideas] [answered]
If you see my personal profile, the Pull Shark achievement only shows "x2" which I suppose means the count of repositories that merged my PRs?? But I know I have at least 3, including [laravel/laravel](https://github.com/laravel/laravel). Anyways, liking this feature!

**Comment by HUMBLEGROUND** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Reply by kuldeeprajput-dev** [reply]
hello bro, have you got the badge or still issue? because i am currently facing this issue, can you help me if your issue resolved ?

<img width="940" height="141" alt="image" src="https://github.com/user-attachments/assets/a49d3639-38a2-40cf-bbf4-06112fe3cd00" />



see my 2 pr merged 



<img width="763" height="669" alt="image" src="https://github.com/user-attachments/assets/0c6ba52f-8756-4a6c-8c41-6366336c9a49" />

see pull shark is not there

**Comment by melvincwng** [comment, marked as answer]
@elidrissidev , the Pull Shark achievement showing "x2" means that you have unlocked the 'Bronze' version of that particular achievement. In short, it means you submitted at least 16 Pull Requests to unlock that badge.

For your reference:
- Merge 2 PR --> unlock basic Pull Shark badge
- Merge at least 16 PR --> unlock 'Bronze' Pull Shark badge (this is where you're at now)
- Merge at least 128 PR --> unlock 'Silver' Pull Shark badge
- Merge at least 1048 PR --> unlock 'Gold' Pull Shark badge

For your reference, you can go to: https://github.com/Schweinepriester/github-profile-achievements for more information

Hope this helps :)

**Reply by elidrissidev** [reply]
Thanks! That clears it up for me.

**Reply by melvincwng** [reply]
Np, glad to have helped!

**Reply by phantomwork** [reply]
Hi, @elidrissidev 
I think you could check the answer so that this question doesn't remain "Unanswered". :)

**Reply by elidrissidev** [reply]
Correct, I just did that :)

**Reply by samueltuoyo15** [reply]
Please some should help me cuz I have 18 pr merged but I didn't got my X2 shark badge I already have the basic but I don't have the bronze why even tho I surpassed the 16 to 18

My Account 
https://github.com/samueltuoyo15

**Reply by kuldeeprajput-dev** [reply]
hello bro, have you got the badge or still issue? because i am currently facing this issue, can you help me if your issue resolved ?

<img width="940" height="141" alt="image" src="https://github.com/user-attachments/assets/a49d3639-38a2-40cf-bbf4-06112fe3cd00" />



see my 2 pr merged 



<img width="763" height="669" alt="image" src="https://github.com/user-attachments/assets/0c6ba52f-8756-4a6c-8c41-6366336c9a49" />

see pull shark is not there

**Reply by samueltuoyo15** [reply]
Yes i got mine

**Reply by samueltuoyo15** [reply]
Just keep building it will surely show up one day

**Reply by kuldeeprajput-dev** [reply]
> Just keep building it will surely show up one day

How much time it is taked in your case

**Reply by samueltuoyo15** [reply]
Some couple months!

**Reply by amrthesha213-netizen** [reply]
It will one two weeks to appear
